src/bridge/test-peer.c

```diff
@@ -470,6 +483,110 @@ test_reopen (TestCase *tc,
   g_free (bridge);
 }
 
+static void
+test_timeout (TestCase *tc,
+             gconstpointer unused)
+{
+  gchar *bridge;
+  GBytes *sent;
+  JsonObject *control;
+  CockpitTransport *other = NULL;
+  gboolean closed = FALSE;
+
+  bridge = g_strdup_printf("{ \"match\": { \"payload\": \"upper\" }, \"timeout\": 1, \"spawn\": [ \"/%s\", \"--upper\", \"--count\" ] }", BUILDDIR "/mock-bridge");
```

Comment:
This is a constant string, so no need for dynamic allocation and printf. (But it doesn't hurt)